tests/test-camptix.php

```diff
@@ -41,13 +41,13 @@ public function test_esc_csv() {
 			'\'@HYPERLINK("http://malicious.example.org/wp-login.php","Please log back in to your account for more.")',
 			"'-2+3+cmd|' /C mstsc'!A0",
 			"'+2+3+cmd|' /C mspaint'!A0",
-			"';2+3+cmd|' /C calc'!A0",
+			";2+3+cmd|' /C calc'!A0",
 
 			// Cells split by delimiters
 			"foo ;'=cmd|' /C SoundRecorder'!A0",
 			"foo\n'-2+3+cmd|' /C explorer'!A0",
-			"'   '-2+3+cmd|' /C notepad'!A0",
-			"' '-2+3+cmd|' /C calc'!A0",
+			"   '-2+3+cmd|' /C notepad'!A0",
+			" '-2+3+cmd|' /C calc'!A0",
 
```

Comment:
Are you 100% sure that all 3 of those are safe? IIRC they were all working payloads, but it may require certain versions of Excel, only work on Windows but not OS X, etc. Here's some details that may be helpful:

https://hackerone.com/reports/151516
https://hackerone.com/reports/160500
https://hackerone.com/reports/160520
https://hackerone.com/reports/164674

I wonder if we're going too far down a rabbit hole here? Are the performance benefits significant enough that they're worth the time involved in refactoring this, and the risk associated with introducing a security hole if we're not super careful?

Reply:
I read up all those reports, and does not look like we need to escape the first char if its delimiter. 
Just escaping the trigger chars when previous character is a delimiter, or when it is the first char of string should be good enough IMO.  
But any case, if you think we should still be escaping delimiter in first char, we can merge this commit https://github.com/Automattic/camptix/commit/1ffff781188972ec00df87532a361656856ecdcc on top of this PR. This has the performance improvement, and also preserve the older behavior.

Reply:
Yeah, I'd prefer to keep the old behavior, just to be safe.